samples/drivers/video/capture_to_lvgl/src/main.c

```diff
@@ -131,11 +131,10 @@ int main(void)
 	display_blanking_off(display_dev);
 
 	const lv_img_dsc_t video_img = {
-		.header.always_zero = 0,
```

Comment:
Should put these changes in a separate commit as they are unrelated. This make the changes easier to be tracked of later.

Reply:
They are in a seperate commit?
https://github.com/zephyrproject-rtos/zephyr/pull/68168/commits/68290e58bbffeb19f81fa4f1f586fb8ce3fda5c6
Or do you mean that the commit title should be adjusted?

Reply:
Alright, I thought that since the goal of the overall commit was to fix the build issue I'd have both in one, given that the PR is already quite hefty in terms of commit count. But sure, can split that.

Reply:
Changed the title and body, is this better?

Reply:
yes, thanks